### PR TITLE
add tests for checking destination namespace

### PR DESCRIPTION
**Pull request by SofiiaZaitseva**

### CODE REVIEW

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts [hunk 1]
```diff
@@ -293,4 +293,160 @@ describe("Connection main actions", () => {
     deleteSource(sourceName);
     deleteDestination(destName);
   });
+
+  it("Saving a connection's destination type with DestinationNamespaceCustomFormat value", () => {
```
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:297]: Please use the actual dropdown option value name, for increasing readability

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts [hunk 2]
```diff
@@ -293,4 +293,160 @@ describe("Connection main actions", () => {
...
+        name: "pokemon",
+      });
+    });
+    checkSuccessResult();
+
+    deleteSource(sourceName);
+    deleteDestination(destName);
+  });
+
+  it("Saving a connection's destination type with DestinationNamespaceSourceFormat value", () => {
```
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:349]: same here

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts [hunk 3]
```diff
@@ -293,4 +293,160 @@ describe("Connection main actions", () => {
...
+        name: "pokemon",
+      });
+    });
+    checkSuccessResult();
+
+    deleteSource(sourceName);
+    deleteDestination(destName);
+  });
+
+  it("Saving a connection's destination type with DestinationNamespaceDefaultFormat value", () => {
```
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:401]: same here

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts (hunk 1 above)
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:297]:
```suggestion
  it("Saving a connection's destination namespace with "Custom format" option", () => {
```

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts (hunk 2 above)
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:349]:
```suggestion
  it("Saving a connection's destination namespace with "Mirror source structure" option", () => {
```

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts (hunk 3 above)
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:401]:
```suggestion
  it("Saving a connection's destination namespace with "Destination default", () => {
```

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts [hunk 4]
```diff
@@ -293,4 +293,160 @@ describe("Connection main actions", () => {
...
+
+    createTestConnection(sourceName, destName);
+
+    goToSourcePage();
+    openSourceDestinationFromGrid(sourceName);
+    openSourceDestinationFromGrid(destName);
+
+    goToReplicationTab();
+
+    selectSchedule("Every hour");
```
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:311]: Can we use the default schedule value?

File: airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts [hunk 5]
```diff
@@ -6,7 +6,7 @@ const databaseInput = "input[name='connectionConfiguration.database']";
 const usernameInput = "input[name='connectionConfiguration.username']";
 const passwordInput = "input[name='connectionConfiguration.password']";
 const pokemonNameInput = "input[name='connectionConfiguration.pokemon_name']";
-const schemaInput = "input[name='connectionConfiguration.schemas']";
+const schemaInput = "#react-select-5-input";
```
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts:9]: Double-check the ID, it looks like the generated one

File: airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts [hunk 6]
```diff
@@ -50,5 +50,13 @@ export const enterSchema = (value: string) => {
   if (!value) {
     return;
   }
-  cy.get(schemaInput).clear().clear().type("{selectAll}{del}").type(value);
+  removeSchema();
```
Comment (dizel852) [inline airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts:53]: Make "enterSchema" function atomic - don't need to remove schema each time if we want to add new schema

**Review by dizel852 — CHANGES_REQUESTED** [review]
Overall - code LGTM
Need to fix some comments

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts (hunk 4 above)
Comment (SofiiaZaitseva) [reply airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:311]: yep, agreed

File: airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts (hunk 6 above)
Comment (SofiiaZaitseva) [reply airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts]: fixed, thanks!

File: airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts (hunk 5 above)
Comment (SofiiaZaitseva) [reply airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts]: fixed, thanks!

**Review by dizel852 — COMMENTED** [review]
Tests and fixes - LGTM, not tested locally
СI tests have passed - I assume everything is ok.

**Review by dizel852 — APPROVED** [review]

**Pull request merged**